prac_07/project_management.py

```diff
@@ -0,0 +1,122 @@
+import csv
+from datetime import datetime
+from project import Project
+
+FILENAME = "projects.txt"
+
+
+def main():
+    projects = load_projects(FILENAME)
+    print(f"Welcome to Pythonic Project Management\nLoaded {len(projects)} projects from {FILENAME}")
+    menu = """
+    - (L)oad projects  
+    - (S)ave projects  
+    - (D)isplay projects  
+    - (F)ilter projects by date
+    - (A)dd new project  
+    - (U)pdate project
+    - (Q)uit
+    """
+
+    choice = ""
```

Comment:
Could refactor this menu so it follows the menu pattern we have in the programming guide.